typegraph/node/src/deps/fromFileUrl.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright Metatype OÜ, licensed under the Mozilla Public License Version 2.0.
+// SPDX-License-Identifier: MPL-2.0
```

Comment:
can you import the license header and rename the file to _import, similar to https://github.com/metatypedev/metatype/blob/main/typegate/engine/src/runtimes/prisma/engine_import.rs#L3-L17 but with the MIT License?